libs/x-content/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -253,6 +255,10 @@ public XContentBuilder startObject() throws IOException {
         return this;
     }
 
+    public XContentBuilder startObject(ParseField field) throws IOException {
```

Comment:
I'm not sure we should add this one. Maybe. But I think it is worth its own little PR if we do.

Reply:
indeed.